packages/fiori/src/ShellBar.js

```diff
@@ -863,12 +926,80 @@ class ShellBar extends UI5Element {
 		return !!this.profile.length;
 	}
 
+	get hasMenuItems() {
+		return this.menuItems.length > 0;
+	}
+
+	get menuBtnHasPopup() {
+		return this.hasMenuItems ? true : undefined;
```

Comment:
"true" seems more appropriate, but if it works this way it's ok

Reply:
It will be cast to string in when it is used in hbs